### ISSUE TITLE
having trouble implementing user auth

### ISSUE THREAD

**Issue by SafeAF** [zayneio/open-flights#11]
love your video tutorials on youtube man. Im stuck at implementing user auth in my version. Could really use some docs or a video on how to implement the user in this app. Really appreciate all the effort!

**Comment by Kirill-Stulov** [comment]
> 
> 
> love your video tutorials on youtube man. Im stuck at implementing user auth in my version. Could really use some docs or a video on how to implement the user in this app. Really appreciate all the effort!

Yeah, would really nice to have it! Maybe in part #5 of that great vid https://youtu.be/iqh9enFWHuY
